src/tracking.c

```diff
@@ -240,13 +244,15 @@ void trackingRememberKeys(client *tracking, client *executing) {
     for(int j = 0; j < numkeys; j++) {
         int idx = keys[j].pos;
         sds sdskey = executing->argv[idx]->ptr;
-        rax *ids = raxFind(TrackingTable,(unsigned char*)sdskey,sdslen(sdskey));
-        if (ids == raxNotFound) {
-            ids = raxNew();
+        void *result;
+        if (!raxFind(TrackingTable,(unsigned char*)sdskey,sdslen(sdskey),&result)) {
+            rax *ids = raxNew();
             int inserted = raxTryInsert(TrackingTable,(unsigned char*)sdskey,
                                         sdslen(sdskey),ids, NULL);
             serverAssert(inserted == 1);
+            result = ids;
         }
+        rax *ids = result;
```

Comment:
Two different `rax *ids` in nested scopes and assignment back and forth is a bit uncomfortable. How about this style:

```suggestion
        rax *ids;
        void *result;
        if (!raxFind(TrackingTable,(unsigned char*)sdskey,sdslen(sdskey),&result)) {
            ids = raxNew();
            int inserted = raxTryInsert(TrackingTable,(unsigned char*)sdskey,
                                        sdslen(sdskey),ids, NULL);
            serverAssert(inserted == 1);
        } else {
            ids = result;
        }
```